pages/api/home/home.tsx

```diff
@@ -179,6 +179,7 @@ const Home = ({
   // CONVERSATION OPERATIONS  --------------------------------------------
 
   const handleNewConversation = () => {
+    const conversations = getConversationHistory();
```

Comment:
Needed to create a block scoped `conversations` because it kept referring to the initial state `conversations` which was empty.